docs/docs/installation/troubleshooting.md

```diff
@@ -48,24 +48,19 @@ More information can be found in the
 
 ## I have pending Pods after Keptn is uninstalled
 
-> **Note** This section particularly affects clusters managed by ArgoCD.
-
-If you have uninstalled Keptn and are now facing issues scheduling or deleting pods, follow these steps:
-
-ArgoCD does not delete various CRDs and webhooks, when uninstalling applications, causing lingering resources.
-
-### For cleanup instructions
-
-1. Install Keptn & CRDs w/ Argo
-2. Uninstall Keptn via Argo
-3. Cluster is frozen, Pods cannot be deleted or scheduled
-
-Trying to schedule a new pod results an `mpod.keptn.sh` error.
-
-Reason: Argo doesn't remove the CRDs and global objects (mutating webhooks) and the
-[default failurePolicy is to fail](https://github.com/keptn/lifecycle-toolkit/blob/650ecba95624ed3dc2bd61bf1f86578f450223a5/operator/config/webhook/manifests.yaml#L17).
-
-to get further information refer to this [issue](https://github.com/keptn/lifecycle-toolkit/issues/1828).
+> **Note** This section particularly affects clusters where
+Keptn was installed via ArgoCD.
+
+If you have uninstalled Keptn, originally installed via ArgoCD
+and are now facing issues scheduling or deleting pods for other applications,
+you probably didn't enable
+[cascading deletion](https://kubernetes.io/docs/concepts/architecture/garbage-collection/#cascading-deletion)
+of the application during installation, which is disabled by default in ArgoCD.
+
+To fix this problem, you need to install Keptn via ArgoCD again, with the use
```

Comment:
Maybe also suggest the alternative that is manually deleting the webhook

Reply:
but what if some other resources are still in place? It's not completely safe and the user may occur other problems then